packages/cli-lib/package.json

```diff
@@ -16,8 +16,9 @@
     "extract-zip": "^1.6.7",
     "findup-sync": "^3.0.0",
     "fs-extra": "^8.1.0",
+    "handlebars": "^4.7.7",
```

Comment:
Are we sure we want to introduce handlebars into the project?

Reply:
Could be worth exploring implementing our own simple version of `handlebars.compile` to avoid the extra package. Here's a relatively simple example of what that could look like https://gomakethings.com/a-vanilla-js-alternative-to-handlebarsjs-and-mustachejs/

Reply:
> One reason handlebars (or similar) is interesting to me is because we've started creating more complex strings that include things like bold or colored text and hyperlinks.

+1 to this. I think it could be valuable to have the translations support bold, colored, links, etc... but I don't think it would be too difficult to spin up our own version of compile that includes that functionality. I didn't dig too deep into the custom implementation though so I could be wrong about that.

Reply:
It definitely makes things easier to be able to use the custom handlers that can be registered with handlebars. There are two currently, and I can see us making more. It allows offloading a lot of the string customization to the `en.lyaml` file, which is nice IMO.

I'm all for looking into writing our own code to achieve this, and the nice thing is we can update the `lang.js` file and it should propagate to all the `i18n` usages. I think it may be better to use handlebars initially and save this for a subsequent PR.

Any objections to going this route? @gcorne @brandenrodgers @anthmatic

Reply:
This has been resolved in https://github.com/HubSpot/hubspot-cli/pull/611